configure.py

```diff
@@ -412,11 +415,12 @@ def configure(args):
     for style in config['themes']:
         configure_style(config, style, str(args.output_dir))
 
-    # create aliases for our light and dark styles to light-blue and dark-blue
-    # FIXME: Invert the order: light should be an alias of light-blue, etc.
-    for theme in ('dark', 'light'):
+    # Create aliases for our light-blue and dark-blue styles to light and dark.
+    # Only create aliases if light-blue and/or dark-blue are to be built.
+    themes = [theme for theme in args.styles if theme in ('dark-blue', 'light-blue')]
+    for theme in themes:
         source = args.output_dir / theme / 'stylesheet.qss'
```

Comment:
This could be improved to use a set but it's fine as-is.

Reply:
Interesting. When you say it could be improved using a set , do you mean as below? If so, what's the benefit to this over using a list comprehension with a tuple? I guess sets are slightly more efficient but is that the only difference in this context? Sorry, probably a noob question!

```python
themes = {theme for theme in args.styles if theme in {'dark-blue', 'light-blue'}}
```

Reply:
I did something like this:

```python
set(args.styles) & {'dark-blue', 'light-blue'}
```

It uses set intersection to grab the overlap between the two. Technically a bit slower but the performance really doesn't matter: it's very easy to read and can be extended pretty well.

Reply:
Ah, good info, thanks. I didn't know set intersection was available in Python so easily. Thanks